src/S3/MultipartUploader.php

```diff
@@ -59,6 +59,7 @@ public function __construct(
         parent::__construct($client, $source, array_change_key_case($config) + [
             'bucket' => null,
             'key'    => null,
+            'exception_class' => 'Aws\S3\Exception\S3MultipartUploadException'
```

Comment:
Maybe my naming here is kind of confusing? What I'm trying to use the class namespace directly like (this)[https://github.com/aws/aws-sdk-php/pull/1140/files#diff-1f37a346f0cdf4ae8ca2d6aaa33bc7a7R127], using `S3MultipartUploadException::class` doesn't seem to work.

Maybe you are suggesting better way that I did get? Could you explain a bit about it? Or perhaps I can do better naming for the parameter there? : )